### ISSUE TITLE
Easy to solve accessibility issues

### ISSUE THREAD

**Issue by drmenzelit** [FreeCAD/FreeCAD-Homepage#212]
Very nice page with a few easy to solve issues:
- Add an `aria-label` or a visually hidden text or an alternative text to the image in the h1
- Change the `<div id="main"...>` to `<main>` element
- Change the link color a little bit to have enough contrast (e.g. #4791fd has a ratio of 4.95 and meet WCAG SC 1.4.3)
- Add an alternative text to the arrow image
- Add an alternative text to the DigitalOcean image/link
- Mark the language flag images as decorative (empty alternative text `alt=""`)
- Add missing h1/h2 on the subpages

Sorry if I'm not following specified guidelines, but the link to "contributing guidelines" leads to a php file without content ...

**Comment by AdarshRawat1** [comment]
@drmenzelit Thanks for the feedback! 

> but the link to "contributing guidelines" leads to a php file without content ..

Which link/button are you referring to ? 
- This one is working 
![image](https://github.com/user-attachments/assets/0cc5ccb1-4ed2-4816-92a6-d6663596ad13)

**Comment by AdarshRawat1** [comment]
@drmenzelit Thanks for the feedback! 

> but the link to "contributing guidelines" leads to a php file without content ..

Could you help me identify which specific button or link you're referring to?
- The one shown here appears to be working:
![image](https://github.com/user-attachments/assets/0cc5ccb1-4ed2-4816-92a6-d6663596ad13)

**Comment by drmenzelit** [comment]
@AdarshRawat1 this one on github:
![grafik](https://github.com/user-attachments/assets/0f1f1136-b0e8-4e73-9690-c5c68c668182)
When creating my first issue here I got a hint here (like a small popup):
![grafik](https://github.com/user-attachments/assets/58084775-5a22-4523-92c9-a165405aecfd)

**Comment by drmenzelit** [comment]
@chennes if I find time in the next days I can check how the website is created and prepare a PR if you like

**Comment by chennes** [comment]
@drmenzelit that would be great -- if you don't have the time just drop a note here and I'll find someone to work on it.

**Comment by Reqrefusion** [comment]
If you're going to bother, I'm not looking. Tag me if something happens.

**Comment by yorikvanhavre** [comment]
@drmenzelit is this issue fully solved with the PR you submitted? I think there are a few things missing, right?

**Comment by yorikvanhavre** [comment]
Okay! Leaving this open then
